start-server.sh

```diff
@@ -22,14 +22,14 @@ nginx
 
 su ${USER}
 
-export OCT_DJANGO_PORT=4010
-export OCT_BASE_DIR="/plugin_data"
-export TRANSFORMERS_CACHE="/models/huggingface"
-export TRANSFORMERS_OFFLINE="0"
-export EASYOCR_MODULE_PATH="/models/easyocr"
-export TESSERACT_PREFIX="/models/tesseract"
-export TESSERACT_ALLOW_DOWNLOAD="true"
-export PADDLEOCR_PREFIX="/models/paddleocr"
+export OCT_DJANGO_PORT=${OCT_DJANGO_PORT:-4010}
+export OCT_BASE_DIR=${OCT_BASE_DIR:-"/plugin_data"}
+export TRANSFORMERS_CACHE=${TRANSFORMERS_CACHE:-"/models/huggingface"}
+export TRANSFORMERS_OFFLINE=${TRANSFORMERS_OFFLINE:-"0"}
+export EASYOCR_MODULE_PATH=${EASYOCR_MODULE_PATH:-"/models/easyocr"}
+export TESSERACT_PREFIX=${TESSERACT_PREFIX:-"/models/tesseract"}
+export TESSERACT_ALLOW_DOWNLOAD=${TESSERACT_ALLOW_DOWNLOAD:-"true"}
+export PADDLEOCR_PREFIX=${PADDLEOCR_PREFIX:-"/models/paddleocr"}
```

Comment:
Allowing a user to customize this could make sense as it would allow for
- A different folder structure in the downloaded models (right now the only customization is what you bind-mount to `/models`
- Allow also the container to be ran in *offline only mode*